pasta_eln/serverActions.py

```diff
@@ -444,6 +444,8 @@ def restoreCouchDB(location:str='', userName:str='', password:str='', fileName:s
     for idx, i in enumerate(possFiles):
       print(f'[{str(idx + 1)}] {i}')
     fileName = input(f'Which file to use for restored? (1-{len(possFiles)}) ')
+    if fileName=='':
+      fileName = '1'
```

Comment:
Why filename is set to string value '1'? What is the need for this`?

Reply:
@jmurugan-fzj  I improved the code with an inline if-else statement